package.json

```diff
@@ -2,13 +2,13 @@
   "devDependencies": {
     "@bazel/bazelisk": "^1.12.0",
     "@bazel/buildifier": "5.1.0",
-    "@bazel/concatjs": "5.5.3",
-    "@bazel/esbuild": "5.5.3",
+    "@bazel/concatjs": "5.7.2",
+    "@bazel/esbuild": "5.7.2",
     "@bazel/ibazel": "^0.16.2",
-    "@bazel/jasmine": "5.5.3",
-    "@bazel/rollup": "5.5.3",
-    "@bazel/terser": "5.5.3",
-    "@bazel/typescript": "5.5.3",
+    "@bazel/jasmine": "5.7.2",
+    "@bazel/rollup": "5.7.2",
+    "@bazel/terser": "5.7.2",
+    "@bazel/typescript": "5.7.2",
```

Comment:
@mattsoulanille I am curious, how did you update all these version? Did you manually find out and update the latest versions one by one here?

Reply:
These versions are always in step with rules_nodejs, so I just update them to the same version number. Usually, this is the latest version, so I just use `yarn upgrade-interactive --latest` to do it for me.

Reply:
Thanks Matt!